pkg/client/block/client.go

```diff
@@ -46,7 +49,19 @@ func NewBlockClient(
 	if err != nil {
 		return nil, err
 	}
-	return &blockClient{eventsReplayClient: client}, nil
+
+	bClient := &blockClient{
```

Comment:
If we rename `blockClient` to `blockReplayClient`, we can name the local variable `blockClient`